envs/example/defaults.yml

```diff
@@ -219,7 +219,7 @@ glance:
   api_workers: 1
   registry_workers: 1
   logging:
-    debug: True
+    debug: False
```

Comment:
The default in the glance role was already false, yet the value was true in glance-api.conf Making this change fixed that.